README.md

```diff
@@ -91,6 +92,32 @@ bwrap --ro-bind /usr /usr \
    /bin/sh
 ```
 
+Sandboxing
+----------
+
+The goal of bubblewrap is to run an application in a sandbox, where it has
+access to less resources.
+
+bubblewrap always creates a new filesystem namespace, and the user can specify
+exactly what parts of the filesystem should be visible in the sandbox.
+Any such directories you specify mounted nodev by default, and can be made readonly.
+
+Additionally you can use these kernel features:
+
+User namespaces: This hides all but the current uid and gid from the
```

Comment:
In this table I'd mention the kernel flags (`CLONE_NEWUSER`).  Also link to e.g. http://linux.die.net/man/2/clone